apps/site/pages/es/about/branding.mdx

```diff
@@ -25,60 +25,56 @@ Créditos a [Angela Angelini](https://www.linkedin.com/in/angeliningl/) por dise
 
 <table>
 <tbody>
-  <tr>
-    <td>
-      <img alt="Logo Horizontal Oscuro de Node.js" src="/static/logos/nodejsDark.svg" className="h-[80px] w-[267px]" width="267" height="80" />
-    </td>
-
-    <td>
-      <img alt="Logo Horizontal Claro de Node.js" src="/static/logos/nodejsLight.svg" className="h-[80px] w-[267px] bg-neutral-950 p-2 dark:bg-transparent" width="267" height="80" />
-    </td>
-
-  </tr>
+<tr>
+<td>
+<img alt="Logo Horizontal Oscuro de Node.js" src="/static/logos/nodejsDark.svg" className="h-[80px] w-[267px]" width="267" height="80" />
+</td>
+
+<td>
+<img alt="Logo Horizontal Claro de Node.js" src="/static/logos/nodejsLight.svg" className="h-[80px] w-[267px] bg-neutral-950 p-2 dark:bg-transparent" width="267" height="80" />
+</td>
+</tr>
 </tbody>
 </table>
 
 ### Logo Apilado de Node.js®
 
 <table>
 <tbody>
-  <tr>
-    <td>
-      <img alt="Logo Apilado Oscuro de Node.js" src="/static/logos/nodejsStackedDark.svg" className="h-[164px] w-[267px]" width="267" height="164" />
-    </td>
-
-    <td>
-      <img alt="Logo Apilado Claro de Node.js" src="/static/logos/nodejsStackedLight.svg" className="h-[164px] w-[267px] rounded bg-neutral-950 p-2 dark:bg-transparent" width="267" height="164" />
-    </td>
-
-  </tr>
-
-  <tr>
-    <td>
-      <img alt="Logo Apilado Negro de Node.js" src="/static/logos/nodejsStackedBlack.svg" />
-    </td>
-
-    <td>
-      <img alt="Logo Apilado Blanco de Node.js" src="/static/logos/nodejsStackedWhite.svg" className="rounded bg-neutral-950 p-2 dark:bg-transparent" />
-    </td>
-
-  </tr>
+<tr>
+<td>
+<img alt="Logo Apilado Oscuro de Node.js" src="/static/logos/nodejsStackedDark.svg" className="h-[164px] w-[267px]" width="267" height="164" />
+</td>
+
+<td>
+<img alt="Logo Apilado Claro de Node.js" src="/static/logos/nodejsStackedLight.svg" className="h-[164px] w-[267px] rounded bg-neutral-950 p-2 dark:bg-transparent" width="267" height="164" />
+</td>
+</tr>
+
+<tr>
+<td>
+<img alt="Logo Apilado Negro de Node.js" src="/static/logos/nodejsStackedBlack.svg" />
+</td>
+
+<td>
+<img alt="Logo Apilado Blanco de Node.js" src="/static/logos/nodejsStackedWhite.svg" className="rounded bg-neutral-950 p-2 dark:bg-transparent" />
+</td>
+</tr>
```

Comment:
this is the thrashing we are trying to avoid - as far as I can tell, its not coming from crowdin - i wrote a short script to do this locally - and confirmed input and output, so my current theory now is that we call format differently. going to try and verify locally again

Reply:
`npm run lint` on the local branch complains about the home pages of each locale - they have unique codeblocks - i will feed that into my program

Reply:
huh not only a lint - a failure 
![image](https://github.com/user-attachments/assets/6f9ffd1f-ae7b-4df2-8dce-858196995e1f)

i bet the indentation is throwing off the markdown parsing